packages/riipen-ui/src/components/Input.jsx

```diff
@@ -204,6 +223,10 @@ const Input = ({
         .warning {
           border-color: ${theme.palette.warning.main};
         }
+
+        .wrapper {
+          position: relative;
```

Comment:
Add position relative so that the start/end position absolute is relative to the wrapper.